npm/private/npm_translate_lock_state.bzl

```diff
@@ -62,12 +54,6 @@ WARNING: `update_pnpm_lock` attribute in `npm_translate_lock(name = "{rctx_name}
     pnpm_lock_exists = is_windows or utils.exists(rctx, label_store.path("pnpm_lock"))
     if pnpm_lock_exists:
         _load_lockfile(priv, rctx, label_store)
-
-    if not priv["patched_dependencies_labels_initialized"]:
-        # If we didn't initialize the patched dependency labels above then try again now. We'll either
-        # read the list of patches from the lock file now that it has been read or, if update_pnpm_lock
-        # is enabled, we can derive the existance of a root package.json by looking for a `.` importer
-        # in the `importers` list.
         _init_patched_dependencies_labels(priv, rctx, label_store)
```

Comment:
Note this is in the `if pnpm_lock_exists` and it was not before. I think that is correct though...